oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/WellknownHandler.java

```diff
@@ -105,9 +106,19 @@ public APIGatewayProxyResponseEvent handleRequest(
                                 providerMetadata.setEndSessionEndpointURI(
                                         buildURI("/logout", baseUrl));
 
+                                providerMetadata =
+                                        OIDCProviderMetadata.parse(
```

Comment:
Is it worth adding this an extra check to the WellknownHandlerTest to see if it is parsed correctly?

Reply:
A ParseException will be thrown if the parse goes wrong, I think we will be fine with this